README.md

```diff
@@ -136,6 +138,34 @@ To get your flow connection names, go to your environment and navigate to _Data
 
 As above, you will need to pass a licensed user's email via runtime settings or environment variables if the Package Deployer is not running in the context of a licensed user. In addition, **the connections passed in need to be owned by the user doing the deployment or impersonated by the deployment**.
 
+### Environment variables
+
+#### Set environment variables
+
+You can set Power App environment variables either through system environment variables (for example, those [exposed on Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#access-variables-through-the-environment) from your variables or variable groups) or through Package Deployer [runtime settings](https://docs.microsoft.com/en-us/power-platform/admin/deploy-packages-using-package-deployer-windows-powershell#use-the-cmdlet-to-deploy-packages).
+
+Environment variables must be prefixed with `PACKAGEDEPLOYER_SETTINGS_ENVVAR_` and followed by the schema name. Similarly, runtime settings must be prefixed with `EnvVar:` and followed by the environment variable schema name. For example, if an enviroment variable schema name was `pdt_testvariable`, this could be set via either of the following:
+
+**Environment variable**
+
+```powershell
+$env:PACKAGEDEPLOYER_SETTINGS_ENVVAR_PDT_TESTVARIABLE = "test_value"
+```
+
+**Runtime setting**
+
+```powershell
+$runtimeSettings = @{ 
+    "EnvVar:pdt_testvariable" = "test_value" 
+}
+
+Import-CrmPackage –CrmConnection $conn –PackageDirectory $packageDir –PackageName Package.dll –RuntimePackageSettings $runtimeSettings
```

Comment:
Not sure if this is correct for the `-RuntimePackageSettings` argument! 

https://docs.microsoft.com/en-us/powershell/module/microsoft.xrm.tooling.packagedeployment/import-crmpackage?view=pa-ps-latest

>Used to pass data at runtime to the package that is being deployed in a key=value format delimited by pipe (|): key1=value1|key2=value2|key3=value3. For example: LCID=1033|SkipChecks=true

I've got a helper script in the Development Hub docs https://github.com/ewingjm/development-hub#deploy-the-package

Reply:
I did think this but I had just copied from the other docs we have... Will correct this 😄